examples/bri-3/src/bri/zeroKnowledgeProof/capabilities/verifyCCSMAnchor/verifyCCSMAnchorCommand.handler.ts

```diff
@@ -0,0 +1,22 @@
+import { CommandHandler, ICommandHandler } from '@nestjs/cqrs';
+import { CCSMAnchorAgent } from '../../agents/ccsmAnchor.agent';
+import { VerifyCCSMAnchorCommand } from './verifyCCSMAnchor.command';
+
+@CommandHandler(VerifyCCSMAnchorCommand)
+export class VerifyCCSMAnchorCommandHandler
```

Comment:
We need to agree on what  this command handler verifies? is it only hash existence? or it is also validity of the hash based on the provided document? 

I would suggest we stick with hash existence verification in this command and open a separate githbu issue to first agree and then potentially implement additional verification of the hash.